io.httpie.HTTPie.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.httpie.HTTPie
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+tags:
+  - proprietary
+command: httpie
+rename-icon: httpie
+rename-desktop-file: httpie.desktop
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+  - name: httpie
+    buildsystem: simple
+    build-commands:
+      - unappimage httpie.AppImage
+      - rm httpie.AppImage
+
+      - install -Dm755 httpie.sh /app/bin/httpie
+      - install -dm755 /app/share/applications
+      - sed -e 's|Exec=AppRun|Exec=/app/bin/httpie|' squashfs-root/httpie.desktop
```

Comment:
don't use used to change the desktop file. use `desktop-file-edit`.

Also Exec should NOT be a absolute path.

Reply:
Done